src/main/java/org/dependencytrack/tasks/TaskScheduler.java

```diff
@@ -22,19 +22,9 @@
 import alpine.event.LdapSyncEvent;
 import alpine.event.framework.Event;
 import alpine.model.ConfigProperty;
+import alpine.model.IConfigProperty.PropertyType;
 import alpine.server.tasks.AlpineTaskScheduler;
-import org.dependencytrack.event.ClearComponentAnalysisCacheEvent;
-import org.dependencytrack.event.DefectDojoUploadEventAbstract;
-import org.dependencytrack.event.FortifySscUploadEventAbstract;
-import org.dependencytrack.event.GitHubAdvisoryMirrorEvent;
-import org.dependencytrack.event.OsvMirrorEvent;
-import org.dependencytrack.event.InternalComponentIdentificationEvent;
-import org.dependencytrack.event.KennaSecurityUploadEventAbstract;
-import org.dependencytrack.event.MetricsUpdateEvent;
-import org.dependencytrack.event.NistMirrorEvent;
-import org.dependencytrack.event.PortfolioVulnerabilityAnalysisEvent;
-import org.dependencytrack.event.RepositoryMetaEvent;
-import org.dependencytrack.event.VulnDbSyncEvent;
+import org.dependencytrack.event.*;
```

Comment:
Please replace with individual class imports. Do not use wildcards.

Reply:
Will do ! I think that one should go also in DEVELOPING.md with a coding style guide

Reply:
Would be even better if we had a way to enforce this sort of code style requirements in CI. [checkstyle](https://checkstyle.sourceforge.io/config_imports.html#AvoidStarImport) can do it, but there may be other options as well.